src/components/FileInfo/FileInfoComponent.tsx

```diff
@@ -19,7 +21,7 @@ export enum FileInfoType {
 @observer
 export class FileInfoComponent extends React.Component<{
     infoTypes: FileInfoType[],
-    HDUOptions?: {HDUList: IOptionProps[], handleSelectedHDUChange: (hdu: string) => void;},
+    HDUOptions?: { HDUList: IOptionProps[], handleSelectedHDUChange: (hdu: string) => void; },
```

Comment:
unnecessary whitespace change. We can change them in a separate issue (e.g. reformatting with prettier)